ui/src/common/components/CitationSummaryTable/CitationSummaryTable.jsx

```diff
@@ -0,0 +1,123 @@
+  render() {
+    const {
+      publishedBucket,
+      citeableBucket,
+      hIndex,
+      loading,
+      error,
+      renderNumberOfCiteablePapers,
+      renderNumberOfPublishedPapers,
+    } = this.props;
+
+    return (
+      <LoadingOrChildren loading={loading}>
+        <ErrorAlertOrChildren error={error}>
+          <div className="__CitationTable__">
+            <table>
+              <tbody>
+                <tr>
+                  <th>
+                    <strong className="f5">Citation Summary</strong>
+                  </th>
+                  <th>
+                    <LabelWithHelp
+                      label="Citeable"
+                      help="Citeable papers have sufficient metadata to reliably track their citations."
+                    />
+                  </th>
+                  <th>
+                    <LabelWithHelp
+                      label="Published"
+                      help={CITABLE_HELP_MESSAGE}
+                    />
+                  </th>
+                </tr>
+                <tr>
+                  <th>Papers</th>
+                  <td>
+                    {citeableBucket.get('doc_count') === 0
+                      ? '0'
+                      : renderNumberOfCiteablePapers(
+                          citeableBucket.get('doc_count')
+                        )}
+                  </td>
+                  <td>
+                    {publishedBucket.get('doc_count') === 0
+                      ? '0'
+                      : renderNumberOfPublishedPapers(
+                          publishedBucket.get('doc_count')
+                        )}
+                  </td>
+                </tr>
+                <tr>
+                  <th>Citations</th>
+                  <td>{citeableBucket.getIn(['citations_count', 'value'])}</td>
+                  <td>{publishedBucket.getIn(['citations_count', 'value'])}</td>
+                </tr>
+                <tr>
+                  <th>h-index</th>
+                  <td>{hIndex.get('all')}</td>
+                  <td>{hIndex.get('published')}</td>
+                </tr>
+                <tr>
+                  <th>Citations/paper (avg)</th>
+                  <td>
+                    {(
+                      citeableBucket.getIn(['average_citations', 'value']) || 0
+                    ).toFixed(1)}
+                  </td>
+                  <td>
+                    {(
+                      publishedBucket.getIn(['average_citations', 'value']) || 0
+                    ).toFixed(1)}
+                  </td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+        </ErrorAlertOrChildren>
+      </LoadingOrChildren>
+    );
+  }
+}
+
+CitationSummaryTable.propTypes = {
+  publishedBucket: PropTypes.instanceOf(Map),
+  citeableBucket: PropTypes.instanceOf(Map),
+  hIndex: PropTypes.instanceOf(Map),
+  renderNumberOfCiteablePapers: PropTypes.func,
+  renderNumberOfPublishedPapers: PropTypes.func,
```

Comment:
Why did you make these as props instead of just putting the NumberOfPublished/CiteableContainer in here? Won't we most probably have the same behavior for the literature search as well? Is there a case where the behavior is different?

Reply:
we usually use `render` methods rather passing down components. That was the only reason.

Reply:
I would pass down the component in this case as I think we will need it the same way for literature search and this will add duplicated code in the search and author detail page

Reply:
well for now it seems to cause duplication of single line functions, but when it gets complicated (and it will), this will be better than passing component that expects certain props..